conf/layer.conf

```diff
@@ -17,6 +17,10 @@ BBFILES += "${LAYERDIR}/recipes-*/*/*.bb ${LAYERDIR}/recipes-*/*/*.bbappend"
 BBFILE_COLLECTIONS += "openrc-layer"
 BBFILE_PATTERN_openrc-layer := "^${LAYERDIR}/"
 
+BBFILES_DYNAMIC += " \
+    meta-virtualization:${LAYERDIR}/recipes-appends/meta-virtualization/*.bbappend \
```

Comment:
Despite what I said in #12, this will have to move to `${LAYERDIR}/dynamic-layers/meta-virtualization/*/*.bbappend` and with your appends moving to `${LAYERDIR}/dynamic-layers/meta-virtualization/docker/`.

Which really points out that I should have put `recipes-appends/oe-core` into `dynamic-layers/oe-core` to begin with.

Reply:
Yes, makes sense. 👍